### ISSUE TITLE
Not Working

### ISSUE THREAD

**Issue by JosephKnight615** [FrostCo/AdvancedProfanityFilter#572]
After getting the extension set up, I made sure it was enabled, but when I go to Youtube it doesn't mute or censor the profanity.

Im using a MacBook Pro 2019, OS 14.6.1 
Safari and Chrome Browser, latest versions

**Comment by richardfrost** [comment]
I'm sorry it isn't working for you @JosephKnight615. To mute audio, you'll need to make sure that you enabled the feature (Options page > Audio tab), and also make sure that the captions are turned on for the YouTube video. For testing, can you try adding a word found in the captions and then see if it gets muted correctly?

Unfortunately there is a problem on YouTube where they choose to censor some of the more severe profanity from their captions/subtitles, which will cause this filter to not be able to mute correctly. If you can share a video (and approximate timestamp) I'm happy to take a quick look at it.

**Comment by JosephKnight615** [comment]
https://youtu.be/vfstrZJZvTY?si=iAV92qS4P0GsvVek&t=10

It's a video on the word Sh^t . It they start using it 12 seconds in.

**Comment by richardfrost** [comment]
Thanks @JosephKnight615. This does look to be one of those examples where the "s" word was not included in the captions. If I had to guess, it was intentional on YouTube's part because it was pretty clear and didn't have a too much background noise or anything. If you add another word, such as `literally`, you should see that it does mute correctly then.

Unfortunately right now there isn't a good way to solve this issue. In the future we'll likely be adding a new feature that will allow you to upload your own captions/subtitles (https://github.com/FrostCo/AdvancedProfanityFilter/issues/500). One of the thoughts there is that people will be able to utilize AI to generate subtitles for a video that includes the profanity so the filter can work.

As you probably know, the filter is only as good as the quality of the captions/subtitles.

**Comment by JosephKnight615** [comment]
Ok , how long do you think that update might be from now?

**Comment by richardfrost** [comment]
@JosephKnight615 im hoping to have it in the next release in the next week or two, but no promises on the exact ETA.

**Comment by han5enar** [comment]
@richardfrost hey I'm just checking if is this still planned? The default YouTube subtitles are so bad...

**Comment by richardfrost** [comment]
@han5enar yes, it is still planned, but I have been very busy and haven't had time to work on it much. It is a high priority though, and its next on my list.
